man/systemd.exec.xml

```diff
@@ -584,7 +584,8 @@
         <varname>BindPaths=</varname> and be careful with <constant>AF_UNIX</constant> file descriptor
         passing for directory file descriptors, as this would permit processes to create files or directories
         owned by the dynamic user/group that are not subject to the lifecycle and access guarantees of the
-        service. Defaults to off.</para></listitem>
+        service. Also note that using this option will stop the unit from being able to publish a new service
+        on D-Bus. Defaults to off.</para></listitem>
```

Comment:
hmm, I'd like to underline that this is a limitation right now, but should go away sooner or later.

i.e. maye:

"Note that this option is currently incompatible with D-Bus policies. A service using this option thus may currently not allocate a D-Bus service name."